stock_picking_product_kit_helper/views/stock_view.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Kitti U. - Ecosoft <carlos.dauden@tecnativa.com>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<record id="view_picking_form" model="ir.ui.view">
+    <field name="name">stock.picking.form</field>
+    <field name="model">stock.picking</field>
+    <field name="inherit_id" ref="stock.view_picking_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="/form/sheet/notebook/page[@name='extra']" position="before">
+            <page string="Product Kit Helper" attrs="{'invisible': [('has_product_kit', '=', False)]}">
```

Comment:
not blocking.
nitpicking : to be mobile friendly, it's better to have short name :  `Kit helper`
 seems sufficient because a kit contains products
Thanks

Reply:
Fixed.